File: examples/eApi v1.9/java/eAPI-v19-connector/src/main/java/cz/monetplus/mips/eapi/v19/connector/entity/MallpayCustomer.java
package cz.monetplus.mips.eapi.v19.connector.entity;

import com.fasterxml.jackson.annotation.JsonInclude;
import lombok.Data;
import lombok.EqualsAndHashCode;

@Data
@EqualsAndHashCode(callSuper = true)
@JsonInclude(value = JsonInclude.Include.NON_NULL)
public class MallpayCustomer extends ApiBase implements Signable {
    private String firstName;
    private String lastName;
    private String fullName;
    private String titleBefore;
    private String titleAfter;
    private String email;
    private String phone;
    private String tin;
    private String vatin;

    @Override
    public String toSign() {
        StringBuilder sb = new StringBuilder();
        add(sb, firstName);
        add(sb, lastName);
        add(sb, fullName);
        add(sb, titleBefore);
        add(sb, titleAfter);
        add(sb, email);
        add(sb, phone);
        add(sb, tin);
        add(sb, vatin);
        deleteLast(sb);
        return sb.toString();
    }

}

File: examples/eApi v1.9/dotNET/CsobGTWDemo/CsobGatewayClientExample/Communication/DataObjects/MallpayCustomer.cs
using System.Text;
using Newtonsoft.Json;

namespace CsobGatewayClientExample.Communication.DataObjects;

public class MallpayCustomer : ApiBase
{
    [JsonProperty("firstName")] public string? FirstName { get; set; }

    [JsonProperty("lastName")] public string? LastName { get; set; }

    [JsonProperty("fullName")] public string? FullName { get; set; }

    [JsonProperty("titleBefore")] public string? TitleBefore { get; set; }

    [JsonProperty("titleAfter")] public string? TitleAfter { get; set; }

    [JsonProperty("email")] public string? Email { get; set; }

    [JsonProperty("phone")] public string? Phone { get; set; }

    [JsonProperty("tin")] public string? Tin { get; set; }

    [JsonProperty("vatin")] public string? Vatin { get; set; }


    public string? ToSign()
    {
        var sb = new StringBuilder();
        Add(sb, FirstName);
        Add(sb, LastName);
        Add(sb, FullName);
        Add(sb, TitleBefore);
        Add(sb, TitleAfter);
        Add(sb, Email);
        Add(sb, Phone);
        Add(sb, Tin);
        Add(sb, Vatin);
        DeleteLast(sb);
        return sb.ToString();
    }
}